package.json

```diff
@@ -22,12 +22,17 @@
   "scripts": {
     "start": "grunt --color --timing",
     "dist": "grunt dist --color",
+    "prebuild": "grunt prebuild:dist --color",
+    "build": "grunt build:dist --color",
+    "dist:teamcity": "npm run lint && npm run test:coverage:teamcity && grunt css:dist --color && grunt docs --color && grunt webpack",
```

Comment:
This dist:teamcity command isn't being used so we should remove it. And also, we have grunt as a task runner for aliasing tasks so we can chain commands in gruntfile rather than here to keep this simpler